app.py

```diff
@@ -177,7 +177,7 @@ def graph_refresh_package_analyzer() -> None:
         "for unanalyzed package versions, packages scheduled: %d",
         _THOTH_GRAPH_REFRESH_EAGER_STOP,
     )
-    packages = _GRAPH_DB.get_unanalyzed_python_package_versions(
+    packages = _GRAPH_DB.get_unanalyzed_python_package_versions_count(
```

Comment:
i think this should be `get_unanalyzed_python_package_versions_all`
https://github.com/thoth-station/storages/blob/a82d77935510cbde18f3a67c70ed8dd47ef12db7/thoth/storages/graph/postgres.py#L2282

Reply:
@pacospace @harshad16 could you please double check, if that is the right APIs to call, I'll make the changes. Thanks!

Reply:
Similar comment as previous please have a look. Have a look at the looping https://github.com/thoth-station/graph-refresh-job/blob/5eba7e5a88486bf91d76cd3cb9675a941e10eac1/app.py#L189

Reply:
WDYT @bissenbay

Reply:
Change it, :+1: !